packages/cpp/src/browser/cpp-keybinding.ts

```diff
@@ -37,6 +37,11 @@ export class CppKeybindingContribution implements KeybindingContribution {
                 commandId: 'switch_source_header',
                 context: this.cppKeybindingContext,
                 keyCode: KeyCode.createKeyCode({ first: Key.KEY_O, modifiers: [Modifier.M3] })
+            },
+            {
+                commandId: 'change_configuration',
+                context: this.cppKeybindingContext,
+                keyCode: KeyCode.createKeyCode({ first: Key.KEY_K, modifiers: [Modifier.M3] })
```

Comment:
Is it necessary to have a key binding for this? Would it be still accessible from the command palette and/or the context menu? If that's the case, I think we should remove the shortcut. If you do please update the commit message, like "Add a way to change compilation database path for clangd"

Reply:
The idea was just to have a command to test if it works correctly. I'm not really convinced that a context menu option for this operation is really worth it for what is, I think, a command that will not be called very often. 

I was thinking that we could aggregate clangd settings (and possibly all C++ extension settings) somewhere else, possibly in another widget/window but I think we will need to discuss this in further details.

Reply:
I think the command palette is ideal for now, so that it doesn't clutter either the context menu or keybinginds. Do you know if it works?

Reply:
It works now. I've removed the keybinding but the command is correctly registered in the command palette.